src/main/drivers/pwm_output_dshot_hal.c

```diff
@@ -331,6 +331,7 @@ bool pwmDshotMotorHardwareConfig(const timerHardware_t *timerHardware, uint8_t m
     case TIM_CHANNEL_2: channel = LL_TIM_CHANNEL_CH2; break;
     case TIM_CHANNEL_3: channel = LL_TIM_CHANNEL_CH3; break;
     case TIM_CHANNEL_4: channel = LL_TIM_CHANNEL_CH4; break;
+    default: break;
```

Comment:
as above. More channels may be added for new timer hardware (IIRC 6 channels are implemented on some timers)

Reply:
Deleted